selfdrive/ui/translations/main_es.ts

```diff
@@ -721,15 +721,15 @@ Esto puede tardar hasta un minuto.</translation>
     <name>Sidebar</name>
     <message>
         <source>CONNECT</source>
-        <translation>CONNECT</translation>
+        <translation>CONECTAR</translation>
```

Comment:
I believe we would want it to stay as CONNECT since that's the proper name of comma's service.

Reply:
That makes sense, I actually had my doubts on weather to change that or not, I basically did the same as the french translation, but maybe that one is mistaken. I'll update that in my other PR.